CMakeLists.txt

```diff
@@ -5,7 +5,14 @@ project(opentracing-cpp)
 # ==============================================================================
 # Version information
 
-set(OPENTRACING_ABI_VERSION "1")
+# Increment ABI version for any ABI-breaking change.
+#
+# Also, whenever the ABI is between versions and in development
+# suffix the ABI version number with "_unstable".
```

Comment:
Might be worth adding this comment to some sort of release documentation. Jaeger repos usually have a `RELEASE.md` for this type of thing.

Reply:
Sounds good. I'll add a `RELEASE.md`.

Reply:
Added.